designer/public/js/CropPrediction.js

```diff
@@ -0,0 +1,793 @@
+$(function() {
```

Comment:
all the code is written wrt NGSI-V1 which is not as per design . Need to update as per the design(protocol independent).